.travis.yml

```diff
@@ -101,26 +100,24 @@ jobs:
           fi
 
     - stage: publish in ter
-      if: tag IS present
+      if: tag IS present AND env(TYPO3_ORG_USERNAME) IS present AND env(TYPO3_ORG_PASSWORD) IS present
```

Comment:
I hope this works by now. Back then when I learned from this Travis feature conjunctions of conditions didn't work.

Reply:
hm, at least it is documented like this